apps/passport-client/src/dispatch.ts

```diff
@@ -942,3 +949,16 @@ async function promptToAgreePrivacyNotice(state: AppState, update: ZuUpdate) {
     });
   }
 }
+
+/**
+ * Merge in a PCD collection from an import of backed-up data.
+ */
+function mergeImport(
+  state: AppState,
+  update: ZuUpdate,
+  collection: PCDCollection,
+  filter: (pcd: PCD) => boolean
+) {
+  state.pcds.merge(collection, { filter, setFolders: true });
+  update({ pcds: state.pcds });
```

Comment:
There needs to be a savePCDs here somewhere so that results persist in local storage.

Reply:
Ah, yes. Added!